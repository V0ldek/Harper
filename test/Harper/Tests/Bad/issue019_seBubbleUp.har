printAndRet :: Integer -> Integer;
printAndRet n = {
  eval printLn n ();
  return n;
};