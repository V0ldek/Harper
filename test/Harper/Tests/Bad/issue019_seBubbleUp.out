Error: cannot match function's `printAndRet` declared type `Integer -> Integer` with its actual type `Integer -> sideeffect -> Integer`.
During evaluation of:
printAndRet n = {
  eval printLn n () ;
  return n ;
}
Located at line 2 column 1

Execution terminated with an error: type error.